### PR TITLE
Lut data range rescale

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
FYI @thewtex

File: Sources/Interaction/UI/VolumeController/index.js [hunk 1]
```diff
@@ -251,6 +259,15 @@ function vtkVolumeController(publicAPI, model) {
     }
   };
 
+  const rescaleColorMap = publicAPI.setRescaleColorMap;
+  publicAPI.setSescaleColorMap = (value) => {
```
Comment (thewtex) [inline Sources/Interaction/UI/VolumeController/index.js]: `setSescaleColorMap`?  What is this function doing?

Comment (jourdain) [reply Sources/Interaction/UI/VolumeController/index.js]: oups, should be `setRescaleColorMap`

Comment (jourdain) [reply Sources/Interaction/UI/VolumeController/index.js]: true/false so either the full data range is used for the color map or only the opacity range.

Comment (thewtex) [reply Sources/Interaction/UI/VolumeController/index.js]: Ah, thanks for the explanation :+1:

Comment (jourdain) [reply Sources/Interaction/UI/VolumeController/index.js]: Do you mind fixing it? thx!

Comment (thewtex) [reply Sources/Interaction/UI/VolumeController/index.js]: Yes, will do :-)

**Pull request merged**